app/phone/src/main/java/dev/jdtech/jellyfin/fragments/SettingsAppearanceFragment.kt

```diff
@@ -19,5 +21,13 @@ class SettingsAppearanceFragment : PreferenceFragmentCompat() {
             }
             true
         }
+
+        findPreference<SwitchPreferenceCompat>("pref_amoled_theme")?.setOnPreferenceChangeListener { _, isAmoledTheme ->
+            when (isAmoledTheme) {
+               true -> requireActivity().recreate()
+               false -> requireActivity().recreate()
+            }
```

Comment:
We can just directly use this, instead of using a switch-case.

```suggestion
            requireActivity().recreate()
```

Reply:
Done, I was using a diffrent approach before and forgot to change this part ☺